lutris/runners/reicast.py

```diff
@@ -40,28 +65,28 @@ def __init__(self, config=None):
                 'option': 'device_id_1',
                 'type': 'choice',
                 'label': 'Joypad 1',
-                'choices': self.get_joypads(),
+                'choices': reicast.get_joypads(),
```

Comment:
Expensive methods should not be called during the init stage, even if this improve the startup time, there is still an underlying issue there. Also there is something weird going on with your setup, get_joypads should be fast (but even when fast, __init__ methods in runners should *not* make any system calls, ever). 
We have to fine a way to lazy load this.